resources/css/plugin.css

```diff
@@ -216,34 +216,48 @@
         display: inline-block;
     }
 
+    h1,h2,h3,h4,h5,h6 {
+        @apply mt-4 mb-4;
+        @apply leading-[1.5];
+    }
+
     h1,
-    h2,
-    h3,
-    h4,
-    h5,
-    h6 {
+    h2{
         font-weight: bold;
     }
 
     h1 {
-        font-size: 1.75rem;
-        line-height: 1.1;
+        font-size: 1.875rem;
```

Comment:
Please use `@apply` and the built-in font sizing scale for Tailwind https://v3.tailwindcss.com/docs/font-size